client/src/components/Modal.tsx

```diff
@@ -63,7 +63,7 @@ export const Modal = (props: ModalProps) => {
           <ModalFooter>
             {buttonsLeft}
             <HStack>
-              <Button colorScheme="blue" onClick={modalProps.onClose}>
+              <Button onClick={modalProps.onClose}>
```

Comment:
```suggestion
              <Button colorScheme="blue" onClick={modalProps.onClose}>
```
And this doesn't need to be changed in such case.

Reply:
you mean - close button, should stay blue, right?

Reply:
Yeah, with green _Change_ that still makes it stand out.